pyunifiprotect/data/devices.py

```diff
@@ -2478,6 +2478,10 @@ def is_temperature_sensor_enabled(self) -> bool:
     def is_humidity_sensor_enabled(self) -> bool:
         return self.mount_type != MountType.LEAK and self.humidity_settings.is_enabled
 
+    @property
+    def is_leak_sensor_enabled(self) -> bool:
+        return self.mount_type == MountType.LEAK
```

Comment:
```suggestion
        return self.mount_type is MountType.LEAK
```

If this is always an enum, identity checks are faster